test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -317,7 +325,7 @@ contract LoansForkTest is LoansTestBase {
         uint userUnderlyingBeforeRoll = pair.underlying.balanceOf(user);
 
         vm.startPrank(user);
-        pair.cashAsset.approve(address(pair.loansContract), loanAmount);
+        pair.cashAsset.approve(address(pair.loansContract), uint(rollFee));
```

Comment:
this will be a huge number if rollFee is negative. Approval only needed here if rollFee is positive

Reply:
will do